app/code/Magento/Cms/Test/Mftf/ActionGroup/AdminNavigateToPageGridActionGroup.xml

```diff
@@ -1,19 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<!--
- /**
-  * Copyright © Magento, Inc. All rights reserved.
-  * See COPYING.txt for license details.
-  */
--->
-
-<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
-    <actionGroup name="AdminNavigateToPageGridActionGroup">
```

Comment:
Unfortunately, we cannot remove action groups since it's backward incompatible. Use `deprecated` attribute instead.

Reply:
**Done**

Reply:
Please, use `<actionGroup name="GoToAttributeGridPageActionGroup" deprecated="reference to the new group">`

Reply:
@sidolov **Done**